packages/ember-engines/index.js

```diff
@@ -20,4 +39,15 @@ module.exports = ManifestGenerator.extend({
     manifest.vendorFiles.push('assets/node-asset-manifest.js');
     return manifest;
   },
+
+  _findHost() {
```

Comment:
It has not been used anywhere. Why we need it?

Reply:
We don't ✂️🙏